kanopi/rooseveltedu
# adding all paragraphs as fields to all content types.

## Description
Post Training: Paragraphs Should be Accessible from All Content Types

## Affected URL

[Review here](http://pr-68-rooseveltedu.pantheonsite.io)

## Related Tickets

* [Post Training: Paragraphs Should be Accessible from All Content Types](https://kanopi.teamwork.com/#/tasks/26024042)


## Steps to Validate
1. Check each of the content types
1. See 3 new fields; Accordion, Basic Content, Call To Action

## Deploy Notes

_Import configurations_
